.azuredevops/pull_request_template/branches/release.md

```diff
@@ -1,5 +1,11 @@
 # {PR title}
 
+<!--
+  In almost all cases, targeting this branch is incorrect. Did you mean to target an internal/release/* branch?
```

Comment:
@wtgodbe @mmitche, I can't think of a reason to put up an AzDO PR targeting a release/* branch but wrote this to cover that case (just in case).

If there's a reason, please let me know here or offline. If you're not sure, please expand the reviewers 😃

If we find there's no reason for such a PR, I'll narrow the template down to just a comment telling people to retarget their PR.

Reply:
I think it's always wrong - any PR merged to azdo's release/x branch will never make its way back to the public release/x branch (there's no mechanism for mirroring in that direction)

Reply:
Missed this when I merged, sorry.

I don't agree w/ the premise @wtgodbe: We merge from release/* into internal/release/* and, eventually, from there back to GitHub release/* branches.

Reply:
Targeting release/* on AzDO is basically always incorrect. We do not merge AzDO's release/* into internal/release/* (only the github copy). The release/* branch is always a direct ff-only mirror from the github version. This means that changes checked into AzDO's release/* will not make it back into public.